api/src/schema/types/I18NBoolean.js

```diff
@@ -0,0 +1,9 @@
+import { GraphQLBoolean } from 'graphql'
+
+export const createI18NBoolean = i18n => {
+  const I18NBoolean = Object.create(GraphQLBoolean)
+  I18NBoolean.description = i18n.t`
+    The 'Boolean' scalar type represents 'true' or 'false'.
+  `
+  return I18NBoolean
+}
```

Comment:
Is this just because it's cleaner syntax, or because it's a convention I wasn't following?

Reply:
This is so these things can be easily pulled into multiple files. It's a little cleaner maybe but I don't love this either. I thinking is that the commits that follow will benefit from this. We'll see...